tasks/files/snapshot.py

```diff
@@ -778,28 +1310,39 @@ def print_result(rc, message):
         "-r",
         "--requiredspace",
         dest="required_space",
+        nargs="?",
         required=False,
-        type=int,  # choices=range(10,100)
         default=20,
         help="percent of required space in the volume group to be reserved for snapshot",
     )
     snapshot_parser.add_argument(
         "-s",
         "--suffix",
         dest="suffix",
+        default=None,
```

Comment:
If you use `default=''` then you can just use the value directly in most cases, and use `len(args.suffix)` without having to check `if args.suffix is None`

Reply:
added issue to track if that is ok with you:
https://github.com/linux-system-roles/snapshot/issues/12